File: dogfood/snippets/C/99_bottles.c
#include <stdio.h>

int main(void)
{
  int n;

  for(n = 99; n > 0; n--) {
    printf(
      "%d bottles of beer on the wall, %d bottles of beer.\n"
      "Take one down, pass it around, %d bottles of beer on the wall\n\n",
      n, n, n - 1);
  }

  return 0;
}

File: dogfood/snippets/Javascript/99_bottles.js
let beer = 99;
while (beer > 0) {
    let verse = `${beer} bottles of beer on the wall, ${beer} bottles of beer.\n` +
        `Take one down, pass it around, ${beer-1} bottles of beer on the wall\n`;

    console.log(verse);
    beer--;
}
